pkgs/by-name/sa/satellite/package.nix

```diff
@@ -0,0 +1,57 @@
+{ lib
+, python3
+, fetchFromGitea
+, gobject-introspection
+, gtk3
+, libhandy
+, modemmanager
+, wrapGAppsHook
+}:
+
+python3.pkgs.buildPythonApplication rec {
```

Comment:
```suggestion
python3Packages.buildPythonApplication rec {
```

every other usage should be changed to use this as well.

Reply:
@SuperSandro2000 why not?

Reply:
Because the two variables are identically here and I didn't want to send the author into another review round.

Reply:
Is `python3Packages` preferred over `python3.pkgs`? If so, I'll use that instead next time.

Reply:
There are discussions to remove python3.pkgs maybe eventually but this will be done by a treewide change. Continue to use what you prefer for now.